eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -276,8 +276,6 @@
     {% if not ez_field_is_empty( content, field ) %}
     <ul {{ block( 'field_attributes' ) }}>
         {% for contentId in field.value.destinationContentIds if parameters.available[contentId] %}
-        {# @todo should be uncommented once HTTP cache bundle is enabled #}
-        {# {{ fos_httpcache_tag('relation-' ~ contentId) }}#}
```

Comment:
Was the intention here to remove the call to `fos_httpcache_tag`?

Reply:
> Was the intention here to remove the call to `fos_httpcache_tag`? At two places below too?

@emodric yes, it was moved to [eZ Platform HTTP Cache bundle](https://github.com/ezsystems/ezplatform-http-cache/blob/7d3b47963393fa25340156ca007c18f820ba1d75/src/Resources/views/fields/content_fields.html.twig).

Reply:
Seems a bit counterproductive to have the override of the entire template in a package most of devs will not think of looking when wanting to override the template (copy pasting the original and changing some things)

Reply:
> Seems a bit counterproductive to have the override of the entire template in a package most of devs will not think of looking when wanting to override the template (copy pasting the original and changing some things)

Hmm, maybe we could add comment here where to look to handle HTTP cache when overriding template.

I believe this was a left-over from times when HTTP cache was a part of Kernel. Right now it's handled by a different package, so having it here is a break on responsibility. 

From practical perspective, if a template with cache-related tags is overriden by Cache bundle, you could disable cache by disabling cache bundles (at least in theory, haven't checked). If you disable them when having the code removed in this PR, you'll get an error trying to display those fields, which means it's not properly decoupled.

Reply:
I'm aware of all that, but:

1) It is really not practical to override these basic templates in other packages without so much as a comment in the overriden template, because developers will need to know that there is an override in place.

2) Syncing changes to the original file could be a nightmare. If you change something here, you need to remember that there is an override in the http cache bundle.

3) For all intents and purposes, eZ Platform requires http cache to work in production. Either Syfmony reverse proxy, or Varnish/Fastly/whatever. So a tiny bit of coupling is okay in my opinion.

4) Why not use twig `extends` a custom Twig block together `parent()` to cut down on copy paste?

Reply:
> 1. It is really not practical to override these basic templates in other packages without so much as a comment in the original template, because developers will need to know that there is an override in place.

I agree, comment would be useful.

> 2. Syncing changes to the original file could be a nightmare. If you change something here, you need to remember that there is an override in the http cache bundle.

Which would be solved by a comment ;)

> 3. For all intents and purposes, eZ Platform requires http cache to work in production. Either Syfmony reverse proxy, or Varnish/Fastly/whatever. So a tiny bit of coupling is okay in my opinion.

This is incorrect from the point of Symfony bundles architecture. Even if something works, it doesn't mean that it is a clean solution. 

For instance, I have feeling that we haven't encountered problems with that Twig function so far, because we haven't covered the templates with Twig integration tests. If we wanted to rectify that in the future, decoupling would be needed anyway.

> 4. Why not use twig `extends` a custom Twig block together `parent()` to cut down on copy paste?

In theory yes, but I don't see a straightforward way to do it. If I get it right, it would be something like:
```twig
{% block ... %}
    {{ fos_httpcache_tag ... }}
    {{ parent() }}
{% endblock ... %}
```
but [usage of `fos_httpcache_tag` in those blocks](https://github.com/ezsystems/ezplatform-http-cache/blob/v1.0.0-beta1/src/Resources/views/fields/content_fields.html.twig) makes it difficult. Some refactoring of core templates would be required, which is always risky and time-consuming.